app/src/main/java/com/example/concordia_campus_guide/Activities/MainActivity.java

```diff
@@ -70,6 +74,40 @@ protected void onCreate(Bundle savedInstanceState) {
         initComponents();
     }
 
+    @Override
+    protected void onResume()
+    {
+        super.onResume();
+
+        SharedPreferences sharedPreferences = getSharedPreferences(ClassConstants.SHARED_PREFERENCES, MODE_PRIVATE);
+        for(CompoundButton toggleButton: toggleButtonAndCorrespondingToggleType.keySet()) {
+            String value = sharedPreferences.getString(toggleButtonAndCorrespondingToggleType.get(toggleButton), ClassConstants.FALSE);
+            if(value.equals(ClassConstants.TRUE))
+                toggleButton.setChecked(true);
+            else
+                toggleButton.setChecked(false);
+        }
```

Comment:
SonarLint is suggesting having the if/else in one line can you change it to avoid code smells

Reply:
yeah i wasn't sure how they wanted it but thanks Tarek, will make the change!